tests/cli_terraform_test.go

```diff
@@ -172,3 +173,56 @@ func runTerraformCleanCommand(t *testing.T, binaryPath string, args ...string) {
 		t.Fatalf("Failed to run terraform clean: %v", stderr.String())
 	}
 }
+
+func TestCLITerraformENV(t *testing.T) {
```

Comment:
Seems like there's nothing this function is doing we cannot move to the yaml test-cases configuration.